pkg/buildkitutil/buildkitutil.go

```diff
@@ -144,8 +130,8 @@ func PingBKDaemon(buildkitHost string) error {
 }
 
 func pingBKDaemon(buildkitHost string) (output string, _ error) {
-	if runtime.GOOS != "linux" && runtime.GOOS != "freebsd" {
-		return "", errors.New("only linux and freebsd are supported")
+	if runtime.GOOS != "linux" && runtime.GOOS != "freebsd" && runtime.GOOS != "windows" {
```

Comment:
Nit: Easier to maintain
```
import "slices"
...
supportedOs := []string{"linux", "freebsd", "windows"}
if !slices.Contains(supportedOs , runtime.GOOS)) {
...
}
```

Reply:
Sadly, this turns out to require Go 1.21 or newer for the "slices" package. I'll open-code slices.Contains for now, and leave a note that we cam replace it with "slices" later.